### ISSUE TITLE
bossplot should use default MJD when possible

### ISSUE THREAD

**Issue by dkirkby** [dkirkby/bossdata#96]
Running `bossplot --plate P` should check what MJD values are possible from this plate using a platelist query.  When only one MJD is possible, it should automatically use this value.  When more than one MJD is possible, it should print the list of valid MJDs and exit.

**Comment by dkirkby** [comment]
This is now implemented and only considers PLATEQUALITY="good" plates.

When only a single MJD is possible:

```
bossdata% bossplot --plate 6641 --verbose --no-display
Plotting combined spectrum for 6641-56383-0030.
...
```

When multiple MJDs are possible:

```
bossdata% bossplot --plate 3647
Plate 3647 has been observed on MJDs 55181,55241,55476,55827,55945,56219,56568,56596.
Select one of these using the --mjd command-line argument.
```

When no MJD is possible (in this the plate was observed but has PLATEQUALITY="bad"):

```
bossdata% bossplot --plate 3523
Plate 3523 has not been observed with good data quality.
```

**Comment by dkirkby** [comment]
Fixed in 0.2.7
